src/site/content/en/blog/how-to-use-local-https/index.md

```diff
@@ -0,0 +1,317 @@
+  certificates!
+- mkcert is a cross-platform tool. Anyone on your team can use it.
+
+Many operating systems may include libraries to produce certificates, such as openssl. Unlike mkcert
+and similar tools, such libraries may not consistently produce correct certificates, may require
+complex commands to be run, and are not necessarily cross-platform.
+{% endAside %}
+
+{% Aside 'gotchas' %}
+The mkcert we're interested in [this one](https://github.com/FiloSottile/mkcert). ❌ Not [this one](https://www.npmjs.com/package/mkcert).
+{% endAside %}
+
+### Caution
+
+{% Banner 'caution' %}
+
+- Never export or share the file `rootCA-key.pem` mkcert creates automatically when you run `mkcert -install`. **An attacker getting hold of this file can create on-path attacks for any site you may be visiting**. They could intercept secure requests from your machine to any site—your bank, healthcare provider, or social networks. If you need to know where `rootCA-key.pem` is located to make sure it's safe, run `mkcert -CAROOT`.
+- Only use mkcert for **development purposes**—and by extension, never ask end-users to run mkcert commands.
+- Development teams: all members of your team should install and run mkcert **separately** (not store and share the CA and certificate).
+
+{% endBanner %}
+
+### Setup
+
+1.  **Install mkcert (only once).**
+
+    Install mkcert by following the [instructions](https://github.com/FiloSottile/mkcert#installation).
+    For example, on macOS:
+
+    ```bash
+    brew install mkcert
+    brew install nss # if you use Firefox
+    ```
+
+1.  **Add mkcert to your local root CAs.**
+
+    In your terminal, run:
+
+    ```bash
+    mkcert -install
+    ```
+
+    This generates a local certificate authority (CA).
+    Your mkcert-generated local CA is only trusted **locally**, on your device.
+
+1.  **Generate a certificate for your site, signed by mkcert.**
+
+    In your terminal, navigate to your site's root directory or whichever directory you'd like the certificates to be located at.
+
+    Then, run:
+
+    ```bash
+    mkcert localhost
+    ```
+
+    Or if you're using a custom hostname like `mysite.example`:
+
+    ```bash
+    mkcert mysite.example
+    ```
+
+    The command above does two things:
+
+    - Generates a certificate for the hostname you've specified
+    - Lets mkcert—that you've added as a local CA in Step 2—sign this certificate.
+
+    Now, your certificate is ready and signed by a certificate authority your browser trusts locally. You're almost done, but your server doesn't know about your certificate yet!
+
+1.  **Configure your server.**
+
+    You now need to tell your server to use HTTPS (since development servers tend to use HTTP by default) and to use the TLS certificate you've just created.
+
+    How to do this exactly depends on your server. A few examples:
+
+    **👩🏻‍💻 With node:**
+
+    `server.js` (replace `{path/to/certificate...}` and `{port}`):
```

Comment:
Please expand a bit on this, for example: "Create a `server.js` file with the following code and replace {path...} and {port} with your specific values. "